content/agent/prometheus.md

```diff
@@ -15,11 +15,52 @@ further_reading:
 
 ## Overview
 
-This page first looks at the `PrometheusCheck` interface, and then proposes a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py).
+This page first looks at the generic `Prometheus` check, the fastest & simpliest way to scrap custom metrics off prometheus endpoints and then dives in the `PrometheusCheck` interface for more advanced usage finishing by an example of writting a simple Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py).
 
-## Prometheus check interface 
+## Generic Prometheus check
 
-`PrometheusCheck` is [a mother class](https://github.com/DataDog/dd-agent/blob/master/checks/prometheus_check.py) providing a structure and some helpers to collect metrics, events, and service checks exposed via Prometheus. Minimal configuration for checks based on this class include:
+Starting with version 6.1 the agent includes a new [Prometheus](https://github.com/DataDog/integrations-core/tree/master/prometheus) check capable of scrapping prometheus endpoints with only a few lines of configuration.
+
+### Configuration
+
+Edit the `prometheus.yaml` file to add your different instances you want to retrieve custom metrics from.
+
+Minimal configuration of an instance includes:
+
+* A `prometheus_url` that points to the metric route (⚠ this has to be unique).
+* A `namespace` that is prepended to all metrics (to avoid metrics name collision).
+* A list of `metrics` that you want to retrieve as custom metrics, for each metric you can either simply add it to the list `- metric_name` or renaming it like `- metric_name: renamed`.
+
+Note: It's possible to use a `*` wildcard such as `- metric*` that would fetch all matching metrics (to use with caution as it can potentially send a lot of custom metrics)
```

Comment:
* Drop the parentheses and just make this two sentences.
* I'm not sure, but perhaps _generate_ would be a better word than "send" here.
* Add a period at the end of this line.
* No need to use the perfect tense in the disclaimer:
> "Use with caution: this can potentially generate a lot of custom metrics!"